src/types/formatted/emailContact.ts

```diff
@@ -1,5 +1,4 @@
 export type EmailContact = {
-  id: string
   label: string
   address: string
 }
```

Comment:
Should this file be deleted? Is `EmailContact` now included in `/formatted/contactInfor.ts`?